Dockerfile

```diff
@@ -7,14 +7,29 @@ FROM debian/snapshot:${debian_snapshot}
 # See https://github.com/hadolint/hadolint/wiki/DL4006
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
+# Uncomment the RUN below if the default snapshot package manager is slow.
+# Please not that this might cause issues and affects reproducible builds,
+# so only use for development.
+# RUN echo \
+#  deb [arch=amd64] http://ukdebian.mirror.anlx.net/debian buster main non-free contrib\
+# > /etc/apt/sources.list
+
+# Getting curl and certificates dependecies.
+RUN apt-get --yes update \
```

Comment:
Why is it a separate `apt-get` call?

Reply:
No good reason. I thought is going to optimise docker runs, but if the execution doesn't complete it doesn't save an intermediate step. I will merge it.